api/extensions/ext_otel.py

```diff
@@ -103,7 +103,7 @@ def setup_context_propagation():
     def shutdown_tracer():
         provider = trace.get_tracer_provider()
         if hasattr(provider, "force_flush"):
-            provider.force_flush()
+            provider.force_flush()  # ty: ignore [call-non-callable]
```

Comment:
I guess add a “is callable” check can fix this.

Reply:
Any suggestion? I've tried `callable` to check it, but never works.